Arrowgene.Ddon.Shared/Entity/PacketStructure/S2CItemAchievementRewardReceiveNtc.cs

```diff
@@ -0,0 +1,56 @@
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure;
+
+/// <summary>
+///     This NTC has to be sent after an achievement's receivable reward has been clicked on in the achievement UI and the
+///     player should unlock a new craft recipe or potentially an arisen profile background.
+/// </summary>
+public class S2CItemAchievementRewardReceiveNtc : IPacketStructure
+{
+    public PacketId Id => PacketId.S2C_ITEM_ACHIEVEMENT_REWARD_RECEIVE_NTC;
```

Comment:
Are you sure this is the name of the NTC?

Reply:
The only NTCs related to achievements from the debug symbols are these here:

S2C_GET_CHARACTER_ACHIEVEMENT_INFO_NTC => no idea how to trigger this and which one it could be.. maybe by clicking on another player?
S2C_ACHIEVEMENT_COMPLETE_NTC => S2C_ACHIEVEMENT_27_8_16_NTC, very obvious via packet dumps
S2C_ACHIEVEMENT_RECEIVABLE_REWARD_NTC => S2C_ACHIEVEMENT_27_9_16_NTC, very obvious via packet dumps

Then as far as items are concerned there no unmapped notices.

So I concluded it's a new NTC since at least according to our packet captures this NTC is only sent exactly between the C2S_ACHIEVEMENT_ACHIEVEMENT_REWARD_RECEIVE_REQ/RES